_tools/linux-aarch64.Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Docker image for building linux-aarch64 packages from other hosts
+#
+# Example, build cpputest:
+# 1. ❯ docker build -t conda-linux-aarch64 -f _tools/linux-aarch64.Dockerfile _tools
```

Comment:
todo docker incantation for enabling foreign architecture

```bash
sudo apt-get install qemu binfmt-support qemu-user-static # Install the qemu packages
docker run --rm --privileged multiarch/qemu-user-static --reset -p yes # This step will execute the registering scripts
```
https://github.com/multiarch/qemu-user-static